package.json

```diff
@@ -6,14 +6,13 @@
   "scripts": {
     "current": "node -e \"console.log(require('./package.json').version)\"",
     "clean": "rm -rf dist",
-    "build": "webpack --config=webpack.config.js --mode=production",
+    "build": "tsc -m es6 --outDir dist",
     "build:clean": "npm-run-all clean build",
     "test": "jest --coverage --verbose --maxWorkers=2",
     "lint": "npm-run-all lint:js lint:ts lint:ts-types",
     "lint:js": "eslint src/**/*.js",
     "lint:ts": "tslint -p ./tsconfig.json",
     "lint:ts-types": "tsc",
-    "watch": "webpack --config=webpack.config.js --mode=development --watch",
```

Comment:
would `tsc --watch` not work here? Or is watching no longer useful?

Reply:
i haven't been using watch, but yea if tsc has a watch i can put it there